assets/stylesheets/sections/_notifications.scss

```diff
@@ -8,6 +8,7 @@
 		right: 0;
 		bottom: 0;
 	min-width: 400px;
+	z-index: z-index( 'root', '#wpnt-notes-panel2' );
```

Comment:
(Note if you do touch this, it's be great if the `.reader-profile` value is moved to the bottom, since it's the highest value).